shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/json/JsonForwardingServerCall.java

```diff
@@ -52,20 +51,18 @@ public JsonForwardingServerCall(final ServerCall<P, P> call) {
     @SuppressWarnings("unchecked")
     @Override
     public void sendMessage(final P message) {
-        String jsonFormat;
-        JsonReply rep;
         try {
             if (message == null) {
                 delegate().sendMessage(null);
                 return;
             }
 
-            jsonFormat = JsonFormat.printer().includingDefaultValueFields().preservingProtoFieldNames()
+            String jsonFormat = JsonFormat.printer().includingDefaultValueFields().preservingProtoFieldNames()
                     .print((MessageOrBuilder) message);
 
-            rep = JsonReply.newBuilder().setMessage(jsonFormat).build();
-            log.debug("begin send json response: {}", jsonFormat);
-            delegate().sendMessage((P) rep);
+            DynamicMessage respMessage = JsonMessage.buildJsonMessage(jsonFormat);
+            log.info("begin send json response");
```

Comment:
Is there any special meaning here?

Reply:
emmm, I wanto print some log to see if the response data is sent.

Reply:
I suggest you can use `debug`.

Reply:
ok~